pkg/crc/constants/constants.go

```diff
@@ -128,9 +114,21 @@ func GetHomeDir() string {
 
 // EnsureBaseDirExists create the ~/.crc dir if its not there
 func EnsureBaseDirExists() error {
-	_, err := os.Stat(CrcBaseDir)
-	if err != nil {
-		return os.Mkdir(CrcBaseDir, 0750)
+	if _, err := os.Stat(CrcBaseDir); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		if err := os.Mkdir(CrcBaseDir, 0750); err != nil {
+			return err
+		}
+	}
+	if _, err := os.Stat(CrcOcBinDir); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		if err := os.MkdirAll(CrcOcBinDir, 0750); err != nil {
+			return err
+		}
```

Comment:
I'd do that in `installSymlinkOrCopy`

Reply:
I'd still do that in `installSymlinkOrCopy` :) Why isn't this also creating `~/.crc/cache`?
This apparently could be simplified as:
```
func EnsureBaseDirectoriesExist() error {
    return os.MkdirAll(CrcOcBinDir, 0750)
}
```

https://golang.org/pkg/os/#MkdirAll
> If path is already a directory, MkdirAll does nothing and returns nil.